### PR TITLE
Allows users to set a username

### LINKED ISSUE #64: Save a username to currentuser

### PR DESCRIPTION
**Pull request by mirrenil**
This PR will:
- include a modal where the user can select a username
- set the displayName to firebase
- toggle the profile image and show the gmail image if it exists or a basic image.

closes #64

### CODE REVIEW

File: src/Pages/ProfilePage.tsx [hunk 1]
```diff
@@ -31,14 +56,56 @@ function ProfilePage() {
...
+              alt="profile picture"
+            />
+          ) : (
+            <img
+              style={{ width: "250px", height: "250px", borderRadius: "50%" }}
+              src="https://thumbs.dreamstime.com/b/man-profile-cartoon-smiling-round-icon-vector-illustration-graphic-design-135443422.jpg"
+              alt="profile picture"
+            />
+          )}
+          <button
```
Comment (liinneea-a) [inline src/Pages/ProfilePage.tsx:72]: The button is invisible for me 

Screenshot 2022-12-01 at 09 46 55

**Review by liinneea-a — CHANGES_REQUESTED** [review]

File: src/Pages/ProfilePage.tsx (hunk 1 above)
Comment (liinneea-a) [inline src/Pages/ProfilePage.tsx:72]: I think this should be more clear. Maybe the button should have 'set display name' text or something?

**Review by liinneea-a — APPROVED** [review]

**Review by AdaHep — APPROVED** [review]
Looks good :)

File: src/Pages/ProfilePage.tsx (hunk 1 above)
Comment (mirrenil) [reply src/Pages/ProfilePage.tsx:72]: Skärmavbild 2022-12-01 kl  09 55 22
 Is this not clear enough?

**Pull request merged**